src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+import java.util.Scanner;
+
+public class Ui {
+
+    private Scanner input = new Scanner(System.in);
+
+    /**
+     * Show welcome message
+     */
+    public String showWelcomeMessage(){
```

Comment:
Consider adding @return String for this function since the return type is not void. I noticed this in other places as well.